Userland/Libraries/LibJS/Runtime/ArrayPrototype.cpp

```diff
@@ -157,7 +158,13 @@ static Object* array_species_create(GlobalObject& global_object, Object& origina
     if (vm.exception())
         return {};
     if (constructor.is_constructor()) {
-        // FIXME: Check if the returned constructor is from another realm, and if so set constructor to undefined
+        auto& con_obj = constructor.as_function();
```

Comment:
Unless the name appears exactly like this in the spec, which it doesn't, we don't use abbreviations (with some rare exceptions). See https://github.com/SerenityOS/serenity/blob/master/Documentation/CodingStyle.md#names

Reply:
Fixed